dd-sdk-android/src/main/kotlin/com/datadog/android/rum/tracking/ActivityViewTrackingStrategy.kt

```diff
@@ -69,7 +69,9 @@ class ActivityViewTrackingStrategy @JvmOverloads constructor(
     }
 
     override fun onActivityPostResumed(activity: Activity) {
-        super.onActivityPostResumed(activity)
+        // this method doesn't call super, because having super call creates a crash
```

Comment:
Wouldn't that break the behavior on recent version of the OS ? Instead maybe we could have an 
 ```
if (Version > KitKat) {
    super.onActivityPostResumed(activity)
    // …
}
```

Reply:
it won't break, since `onActivityPostResumed` in `super` is empty. Version check doesn't help as I've stated in the PR description.

Reply:
Yes, forward compatibility can be a concern. I'm not sure if `ActivityViewTrackingStrategyKitKat` would work though, because it means that both `ActivityViewTrackingStrategyKitKat` and `ActivityViewTrackingStrategyLollipop` will be referenced from the same class in the `import` statement, and it means `DexVerifier` will still go through the class with `super` call during class load process.